components/question/RangeSlider/index.jsx

```diff
@@ -0,0 +1,62 @@
+import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+
+import { buildSurveyQuestionsWithAnswers } from '../Details/buildSurveyQuestionsWithAnswers';
+import styles from './index.module.scss';
+
+export const ACTIVE_SLIDER_BACKGROUND = 'rgba(255, 255, 255)';
+export const INACTIVE_SLIDER_BACKGROUND = 'rgba(255, 255, 255, 0.18)';
+
+export const rangeSliderTestIds = {
+  base: 'range-slider',
+  input: 'range-slider__input',
+};
+
+const RangeSlider = ({ id, answers, setSurveyQuestionsWithAnswers }) => {
+  const [currentValue, setCurrentValue] = useState(-1);
+  const minValue = 0;
+  const maxValue = answers.length - 1;
+  const progress = `${(currentValue / maxValue) * 100}%`;
+
+  useEffect(() => {
+    setCurrentValue(-1);
+  }, [id]);
+
+  const onRangeValueChange = value => {
+    const surveyQuestionsWithAnswers = buildSurveyQuestionsWithAnswers(id, answers[value].id);
+    setCurrentValue(value);
+    setSurveyQuestionsWithAnswers(prevSurveyQuestionWithAnswer => {
+      let newSurveyQuestionWithAnswer = prevSurveyQuestionWithAnswer.slice();
+      newSurveyQuestionWithAnswer = newSurveyQuestionWithAnswer.filter(response => response.id !== id);
+
+      return [...newSurveyQuestionWithAnswer, surveyQuestionsWithAnswers];
+    });
+  };
+
+  const inputStyle = {
+    background: `linear-gradient(to right, ${ACTIVE_SLIDER_BACKGROUND} 0% ${progress}, ${INACTIVE_SLIDER_BACKGROUND} ${progress} 100%)`,
+  };
+
+  return (
+    <div className="h-14 flex items-center mt-16" data-test-id={rangeSliderTestIds.base}>
```

Comment:
How about adding the `help_text` as `title` and/or `aria-label` for the slider?
![Screen Shot 2565-05-19 at 11 33 06](https://user-images.githubusercontent.com/1772999/169205698-adfce927-ca1b-4d5d-95bb-96af99a6cf50.png)

Reply:
sure, done in 95ba02c628e6a3a0715a5157d5c16bfdbc2dfb17, Thanks!